### PR TITLE
Theta 0 bugfix

### PR DESCRIPTION
**Pull request by Sosnowsky**

### CODE REVIEW

File: blobmodel/blobs.py [hunk 1]
```diff
@@ -71,20 +71,19 @@ def discretize_blob(
...
-        if np.sin(self.theta) == 0:
-            x_border = Ly - self.pos_y
-            adjusted_Ly = Ly
+        if self.theta == 0:
+            number_of_y_propagations = 0
         else:
             x_border = (Ly - self.pos_y) / np.sin(self.theta)
             adjusted_Ly = Ly / np.sin(self.theta)
-        if type(t) in [int, float]:
             # t has dimensionality = 0, used for testing
```
Comment (gregordecristoforo) [inline blobmodel/blobs.py:79]: In the current form it isn't obvious where this comment is referring to. We should make clear this comment refers to the if type(t) line

File: tests/test_blob.py [hunk 2]
```diff
@@ -75,6 +75,37 @@ def test_single_point():
...
+    x = np.arange(0, 10, 0.1)
+    y = 0
+
+    t = np.array(2)
+    mesh_x, mesh_y = np.meshgrid(x, y)
+    blob_values = blob_sp.discretize_blob(
+        x=mesh_x, y=mesh_y, t=t, periodic_y=True, Ly=1
+    )
+
+    maxx = np.max(blob_values)
```
Comment (gregordecristoforo) [inline tests/test_blob.py:102]: This isn't deterministic is it? Would it be hard to determine the amplitude from the input parameters directly?

**Review by gregordecristoforo — CHANGES_REQUESTED** [review]

File: tests/test_blob.py (hunk 2 above)
Comment (gregordecristoforo) [reply tests/test_blob.py:102]: Ok, maybe one can just mention it in a comment then

File: blobmodel/blobs.py (hunk 1 above)
Comment (Sosnowsky) [reply blobmodel/blobs.py]: chnaged

**Review by gregordecristoforo — APPROVED** [review]

**Pull request merged**